src/maliput_malidrive/builder/road_curve_factory.h

```diff
@@ -227,6 +231,7 @@ class RoadCurveFactoryBase {
   ///        than @p p0.
   /// @param assert_continuity If true, C1 continuity is assert when function describing the lane width is built.
   ///                          Otherwise only warning messages are printed.
+  /// @param adapt_lane_widths If true, lane widths that don't comply with G1 continuity are adapted so they do.
```

Comment:
nit: Switch it to C1 as in this case functions do not represent a surface or volume.

Reply:
Done.